baseline/OSPS-BR.yaml

```diff
@@ -12,17 +12,17 @@ criteria:
     maturity_level: 1
     criterion: |
       The project's build and release pipelines
-      MUST NOT permit arbitrary input that allows
+      MUST NOT permit untrusted input that allows
       access to privileged resources.
     rationale: |
       Reduce the risk of code injection or other
       security vulnerabilities in the project's
       build and release by preventing untrusted input
-      to access privileged resources (code execution,
-      secret exfiltration, etc.)
+      to access privileged resources
+      (secret exfiltration, final release, etc.)
     details: |
       Ensure that any build and release pipeline actions
```

Comment:
```suggestion
      Ensure that any integration or release pipeline actions
```

I feel like continuous integration (particularly pre-merge pipelines) are particular targets here.  I feel like we should call them out explicitly.